### ISSUE TITLE
arm/aarch64 build fails in step 9 / Generate deltas

### ISSUE THREAD

**Issue by eszlari** [flathub/org.kicad_pcb.KiCad#19]
```
9 Generate deltas

stdio
view all 15 lines

flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo
 in dir /srv/buildbot/worker/build-aarch64-5/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo'
 using PTY: False
Updating appstream branch
Generating static deltas
Generating delta: appstream2/aarch64 (520f66c1a8)
Generating delta: app/org.kicad_pcb.KiCad/aarch64/test (6583cbd95d)
Generating delta: runtime/org.kicad_pcb.KiCad.Locale/aarch64/test (bbff9ad318)
command timed out: 1200 seconds without output running b'flatpak build-update-repo --generate-static-deltas --static-delta-ignore-ref=*.Debug  --static-delta-ignore-ref=*.Sources repo', attempting to kill
process killed by signal 9
program finished with exit code -1
elapsedTime=1200.129813
```

https://flathub.org/builds/#/builders/32/builds/6733

**Comment by jmaibaum** [comment]
@eszlari I don't know enough about what you had been observing here, but it looks strikingly similar to some of the timeouts I have seen on x86_64 test builds in various other PRs (see for example [this recent build log](https://github.com/flathub/org.kicad_pcb.KiCad/pull/33#issuecomment-632640137)). They look rather random to me.

I am planning a round of dependency updates once #33 is merged, and while tackling these, I wanted to check if we really need to keep out ARM users from using this flatpak. Or do you have more info on what went wrong here?

**Comment by eszlari** [comment]
I guess it's the size. KiCad is AFAIK the biggest flatpak on Flathub:
```
flatpak remote-info flathub org.kicad_pcb.KiCad

KiCad - EDA Suite

        ID: org.kicad_pcb.KiCad
       Ref: app/org.kicad_pcb.KiCad/x86_64/stable
      Arch: x86_64
    Branch: stable
   Version: 5.1.6
   License: AGPL-3.0-or-later
Collection: org.flathub.Stable
  Download: 1.0 GB
 Installed: 5.9 GB
   Runtime: org.freedesktop.Platform/x86_64/19.08

       Sdk: org.freedesktop.Sdk/x86_64/19.08
    Commit: 1aed546d717276363aedf7cc07567c36bb7c77aec8f1e7e49afa17e0d7a7bef4
    Parent: 90c7c65e0f69775d72c2df77fb5ca05ea05681f1cd570374468fc918f0131654
   Subject: Fix cmake-ninja build (4dce26fd)
      Date: 2020-05-21 18:52:24 +0000
```
Especially when it comes to the installed size, AFAIK there is nothing that comes close (compare to LibreOffice, FreeCAD, Openarena etc).

Maybe when https://github.com/flathub/org.kicad_pcb.KiCad/issues/32 is implemented, we will see fewer build failures.
